docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -333,3 +333,57 @@ Mouse event | Touch event
 `mousemove` (when hovering or dragging) | `touchmove` (dragging only)
 `mousedown` | `touchstart`
 `mouseup`   | `touchend`
+
+## Examples
+
+### Download Files in IE
+
+TestCafe prevents native browser dialogs, including those before file download. In Internet Explorer, however, you should disable this dialog in browser settings, because it cannot be dismissed programmatically.
+
+The following example shows how to download the file once you disable the dialog:
+
+```html
+<body>
+    <form method="get" action="./src/file.zip">
+        <button id="downloadButton" type="submit">Download!</button>
+     </form>
+</body>
+```
+
+This sample page includes a button that downloads a file from the `/src` folder on the server. To obtain the file, use a [RequestHook](../../reference/test-api/requesthook/README.md):
```

Comment:
If the goal is getting the content of the downloaded file, use the built-in RequestLogger class.

Reply:
reworked